### PR TITLE
fix needed for cheyenne lmod update

### PR DESCRIPTION
**Pull request by jkbk2004**
This fix addresses the issue [#1470](https://github.com/ufs-community/ufs-weather-model/issues/1470)

### CODE REVIEW

File: tests/fv3_conf/fv3_qsub.IN_cheyenne [hunk 1]
```diff
@@ -13,7 +13,9 @@ echo -n " $( date +%s )," >  job_timestamp.txt
 set +x
 MACHINE_ID=cheyenne
 source ./module-setup.sh
-module use $( pwd -P )
+mkdir makefiles
```
Comment (DusanJovic-NOAA) [inline tests/fv3_conf/fv3_qsub.IN_cheyenne]: Directory should be named `modulefiles` not `makefiles`

Comment (jkbk2004) [reply tests/fv3_conf/fv3_qsub.IN_cheyenne]: oops! fixed it

**Comment by jkbk2004** [comment]
@MatthewMasarik-NOAA can you merge in this cheyenne fix?

**Comment by MatthewMasarik-NOAA** [comment]
> @MatthewMasarik-NOAA can you merge in this cheyenne fix?

@jkbk2004, sure thing.  sorry for the delay, in a meeting but will push in ~15 mins.

**Comment by MatthewMasarik-NOAA** [comment]
@jkbk2004, done.

**Pull request merged**